app/views/cases/offender_sar/_date_received_step.html.slim

```diff
@@ -9,9 +9,9 @@
 = form_for @case, url: case_create_action(@case.object), as: :offender_sar do |f|
 
   .form-group
-    = f.gov_uk_date_field :received_date, { legend_text: t('cases.new.date_sar_received'),
-        form_hint_text: t('cases.new.date_sar_received_copy'),
-        today_button: {class: ''}}
+    = f.gov_uk_date_field :received_date, { legend_text: '',
+      form_hint_text: t('cases.new.date_sar_received_copy'),
+      today_button: {class: ''}}
```

Comment:
Formatting is a bit off. Try this:
```
= f.gov_uk_date_field :received_date, { 
    legend_text: '', 
    form_hint_text: t('cases.new.date_sar_received_copy'),
    today_button: { class: '' }
  }
```

If `class` is being set to `''` then is it still required?

Reply:
Yes might as well refactor this while visiting. Done!

Reply:
OK, so we need the class so it doesn't render it's own class (button) and change the look from link to button, even though it's a link.
Also need backslashes to allow the braces to be on separate lines